Assets/Scripts/с#/Exercise7/Game7Object2.cs

```diff
@@ -0,0 +1,14 @@
+using UnityEngine;
+
+public class Game7Object2 : MonoBehaviour
+{
+    [SerializeField] private Space5OnOff _space5OnOff;
+
+    private void Update()
+    {
+        var obj = FindObjectOfType<Space5OnOff>();
```

Comment:
cache
The operations of obtaining components are heavy, you do not need to receive a component with each update